eng/pipelines/runtime.yml

```diff
@@ -493,8 +493,26 @@ extends:
           jobParameters:
             testScope: innerloop
             nameSuffix: CoreCLR
-            buildArgs: -s clr+libs+host -c $(_BuildConfig) -rc Checked
+            buildArgs: -s clr+libs+host -c $(_BuildConfig) -rc Checked -p:StageOneBuild=true
             timeoutInMinutes: 120
+            postBuildSteps:
+              - script: |
+                  echo Running $(Build.SourcesDirectory)/build$(scriptExt)  \
+                    ${{ variables.debugOnPrReleaseOnRolling }}              \
+                    -s clr.tools+packs -rc Checked -cross                   \
+                    -os $(osGroup)                                          \
+                    -a $(archType)                                          \
+                    -c $(_BuildConfig)                                      \
+                    -p:StageTwoBuild=true
```

Comment:
This echo feels a bit weird to have here.

Reply:
It was added to show the command. Apparently logs don't capture postBuildSteps/script command without it.